specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2018-06-01-preview/apimapis.json

```diff
@@ -3526,7 +3536,10 @@
             "swagger-json",
             "swagger-link-json",
             "wsdl",
-            "wsdl-link"
+            "wsdl-link",
+            "openapi",
+            "openapi+json",
```

Comment:
should it be openapi-json, like other enums

Reply:
No, a proper MIME suffix  for a underlying format  is with "+"